docs/introduction/flows.md

```diff
@@ -59,7 +59,9 @@ All flows use an action property to designate what the flow is doing and how it
 * vf:pass - possible output of a testing or reviewing process, indicating the resource passed, the same resource will appear in input with *vf:accept* verb
 * vf:fail - possible output of a testing or reviewing process, indicating the resource failed, the same resource will appear in input with *vf:accept* verb
 * vf:service - new service produced and delivered (a service implies that an agent actively receives the service)
-* vf:transfer - give rights and/or possession of a resource from one agent to another
+* vf:transfer-accounting - give full (in the human realm) rights and responsibilities to another agent, without transferring physical custody
```

Comment:
Will people understand transfer-accounting? I understand not wanting to say "transfer-ownership" but maybe "transfer-rights"? Or "transfer-all-rights"? 
(But I like the description.)

Reply:
But I could live happily with transfer-accounting.